### DISCUSSION TITLE
PHP packages 8.3.9

### DISCUSSION THREAD

**Discussion by Jir4** [oerdnj/deb.sury.org#2179] [General]
Hi, 

Version [8.3.9](https://www.php.net/ChangeLog-8.php) of PHP has been released, is there a plan to generate those packages ?

**Comment by oerdnj** [comment]
Could you please follow the template, so I at least know what repository you are talking about?  It's the information you have already available, but if you write this just like this ^^^, I need to got check all of them.

(I forgot to click the Build button for Debian, so it's building now...)

**Comment by Jir4** [comment]
I didn't see any template in `discussions` sorry. It's for the Debian repo yes. Thank you for your responsiveness, and all your work.

**Reply by oerdnj** [reply]
My bad, I haven't realised this is a discussion - then please open a bug report next time and there is a template.
